File: app/src/main/java/com/example/exoesqueletov1/fragments/WelcomeFragment.java
package com.example.exoesqueletov1.fragments;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.view.animation.Animation;
import android.view.animation.AnimationUtils;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import com.example.exoesqueletov1.R;

public class WelcomeFragment extends Fragment {

    private View view;

    public WelcomeFragment() { }

    @Nullable
    @Override
    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {
        view = inflater.inflate(R.layout.fragment_welcome, container, false);
        return view;
    }

    @Override
    public void onStart() {
        super.onStart();
        TextView t1 = view.findViewById(R.id.textHola);
        TextView t2 = view.findViewById(R.id.textVamos);
        Animation animation = AnimationUtils.loadAnimation(getContext(), R.anim.transition);
        t1.setAnimation(animation);
        t2.setAnimation(animation);
    }
}

File: app/src/main/java/com/example/exoesqueletov1/ui/fragments/WelcomeFragment.kt
package com.example.exoesqueletov1.ui.fragments

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.view.animation.AnimationUtils
import androidx.fragment.app.Fragment
import com.example.exoesqueletov1.R
import com.example.exoesqueletov1.data.firebase.Constants
import com.example.exoesqueletov1.databinding.FragmentWelcomeBinding

class WelcomeFragment(private val navigation: (Constants.SingInPagerNavigation) -> Unit) :
    Fragment() {

    private lateinit var binding: FragmentWelcomeBinding

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        binding = FragmentWelcomeBinding.inflate(layoutInflater, container, false)
        return binding.root
    }

    override fun onStart() {
        super.onStart()
        val animation = AnimationUtils.loadAnimation(context, R.anim.transition)
        binding.textHola.animation = animation
        binding.textVamos.animation = animation
        binding.buttonGoToLogin.setOnClickListener {
            navigation.invoke(Constants.SingInPagerNavigation.Login)
        }
        binding.buttonDoYouHaveAccount.setOnClickListener {
            navigation.invoke(Constants.SingInPagerNavigation.SingIn)
        }
    }
}